codegen/smithy-aws-swift-codegen/src/test/kotlin/software/amazon/smithy/aws/swift/codegen/AWSSigningMiddlewareTests.kt

```diff
@@ -73,122 +76,164 @@ class AWSSigningMiddlewareTests {
     }
 
     @Test
-    fun `renderSigningMiddleware unsignedBody true`() {
```

Comment:
This and the other tests below were refactored to get a lot of duplicate logic out of them.